MSMS/scripts/msmsconsensus.R

```diff
@@ -0,0 +1,148 @@
+
+library(tidyverse)
+init_dat <- read_csv(r"(MSMS\data_processed\Ingalls_Lab_Standards_MSMS.csv)")
```

Comment:
Just checking, is that "r" a typo?

Reply:
Nope! String literal so I didn't have to escape the backslashes: https://mpopov.com/blog/2020/05/22/strings-in-r-4.x/